WordPress/src/main/res/values/strings.xml

```diff
@@ -1786,6 +1786,12 @@
     <string name="plugin_inactive">Inactive</string>
     <string name="plugin_autoupdates_on">Autoupdates on</string>
     <string name="plugin_autoupdates_off">Autoupdates off</string>
+    <string name="plugin_detail_label_state_active">Active</string>
```

Comment:
Since this string has the same meaning as `plugin_active` we could reference that here so both strings don't need translating.

Reply:
Different languages can have different translations since one of them is the state and the other one is label, so I think it's important to separate them.